composer.json

```
@@ -0,0 +1,7 @@
+{
+    "require": {
+	"php": "^7.1",
+        "ext-mbstring": "*",
+	"ext-sqlite3": "*"
```

```suggestion
	"ext-sqlite3": "*",
	"ext-curl": "*",
	"ext-openssl": "*",
	"ext-libxml": "*",
	"ext-simplexml": "*",
	"ext-json": "*"
```

I didn't test it, but it *should work*™